### ISSUE TITLE
Add wikidata description to dropdown list items

### ISSUE THREAD

**Issue by boothym** [openstreetmap/iD#7460]
I was adding wikidata to an object, however when doing so there were a lot of items with the same name - meaning I had to mouseover each one to get the description to appear.

If there is more than one wikidata item of the same name, could the description be added so users can differentiate between them? E.g. "Green Hill Q3314551 - Census-designated place in Wilson County Tennessee"

![Untitled-1](https://user-images.githubusercontent.com/16965772/77479649-1a6f6580-6e18-11ea-84f2-0ea1ce5a7dc4.jpg)

Would it even be possible to prioritise the geographically closest items in the list?

**Comment by quincylvania** [comment]
@boothym I agree it'd be great to show the descriptions too, but no other comboboxes have subtitles so we'd have to build out this UI. I'd rather not just have one big long string.

> Would it even be possible to prioritise the geographically closest items in the list?

Interested idea! This is probably doable. We could also theoretically filter the list to just geographic entities, but this is a bit risky since not everything on wikidata is properly categorized.

**Comment by boothym** [comment]
In JOSM, the wiki plugin can download elements either in the viewport or from a certain category.

The object I was editing was a peak, don't know if the wikidata api response supplies a catergory which could also help in terms of reducing the number of wikidata items?

![wiki](https://user-images.githubusercontent.com/16965772/77595823-24fe2d80-6ef2-11ea-8f5f-66c7a3af8f65.jpg)

**Comment by 1ec5** [comment]
We could also display a thumbnail image alongside each item, to help the user recognize the correct value more easily: https://github.com/openstreetmap/iD/issues/5566#issuecomment-445292438.

**Comment by Sirorezka** [comment]
I would like to upvote this issue. It is very hard to link with correct wikidata object if you don't see item description. Basically by definition on Wikidata Description is part of object identification. See how it is defined:

> The description on a Wikidata entry is a short phrase designed to disambiguate **items** with the same or similar **labels**. A **description** does not need to be unique; multiple items can have the same description, however **no two items can have both the same label and the same description**.

from: [https://www.wikidata.org/wiki/Help:Description](https://www.wikidata.org/wiki/Help:Description)

Check how item search works on wikidata site. It always includes item description:
![image](https://github.com/openstreetmap/iD/assets/10659800/365fd9be-0ac3-48b5-85cd-4f32d3b335a8)

**Comment by k-yle** [comment]
this was done in #12436
